README.md

```diff
@@ -19,23 +19,24 @@ adop-nginx provides Nginx with LDAP support. Nginx is a high performance reverse
       $ docker run --name <your-container-name> -dt \
         -v /resources/configuration/:/etc/nginx/:ro \
         -v /resources/release_note:/usr/share/nginx/html/:ro \
-        -v /var/log:/var/log 
+        -v /var/log:/var/log
         -p 443:443 \
         -p 80:80 \
         accenture/adop-nginx:VERSION
-        
+
 ## Configuration
 
 The nginx configuration is externalised and stored the 'resources' directory.
 
 Runtime configuration can be provided using environment variables:
 
 * LDAP_SERVER, the LDPA URI, i.e. ldap-host:389
+* LDAP_PROTOCOL, ldap(default) or ldaps
```

Comment:
Can we please add some description for this parameter along with the default value?